In [23]:
from scipy import stats

In [24]:
upper = 185.4 # 6'1" = 185.42
lower = 177.8 # 5'10" = 177.8cm


In [25]:
percentile_upper = stats.norm.cdf(upper , loc=178, scale= 7.7) # Percentage of males under 6'1"
percentile_lower = stats.norm.cdf(lower , loc=178, scale= 7.7) # Percentage of males under 5'10"
bmg_eligible_percentage = percentile_upper - percentile_lower
bmg_eligible_percentage

0.3420946829459531

The percentage of U.S. males between 5'10" and 6'1" is roughly 34% 